BrainPortal/db/migrate/20220905142228_remove_old_remote_ressources_attributes.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveOldRemoteRessourcesAttributes < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :remote_resources, :nh_email_delivery_options, :text,    :after => :nh_system_from_email
```

Comment:
I don't think you need to provide type information for the remote_column operation.

See for instance

https://github.com/aces/cbrain/blob/master/BrainPortal/db/migrate/20170418124708_remove_docker_present_and_singularty_present_from_remote_resources.rb

Reply:
I think I need to since is not `def down` but `def change` that I use. 

https://www.rubyinrails.com/2014/05/14/rails-migration-change-vs-up-down-methods/